plugins/inputs/ecs/stats.go

```diff
@@ -0,0 +1,310 @@
+package ecs
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs/docker"
+)
+
+func parseTaskStats(t Task, acc telegraf.Accumulator) {
```

Comment:
I don't see a real need for this function, as `accTask` is the only caller and calling this is the only thing it does. Put this logic into `accTask`